CHANGELOG.md

```diff
@@ -39,6 +39,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Set IUTF8 termios flag for improved UTF8 input support
 - Dragging files into terminal now adds a space after each path
 - Default binding replacement conditions
+- Selection is now cleared if clear line or clear screen escapes are clearing the content behind it
```

Comment:
That's actually not entirely true, the original problem was that selection was persistent if you clear_line behind it, so the wording could be a bit hard, since in one case it was always cleared, but in the other it was never cleared, while now it's cleared in both cases, if something was changed behind it.